libr/anal/p/anal_sparc_cs.c

```diff
@@ -119,10 +119,6 @@ static int analop(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len, RAn
 	cs_insn *insn;
 	int n;
 
-	if (!a->config->big_endian) {
```

Comment:
reverting this may fix the test. and im fine to merge like this if tests are passing, but it will be good to handle big endian sparc too. will do that in another pr